.editorconfig

```diff
@@ -0,0 +1,20 @@
+# editorconfig.org
+root = true
+
+[*]
+indent_style = space
```

Comment:
The only filetype that uses tabs is php, which is defined below ;) json, xml, yml and md all use spaces.